tests/shared/core/training_data/story_writer/test_markdown_story_writer.py

```diff
@@ -17,4 +17,5 @@ async def test_tracker_dump_e2e_story(default_agent: Agent):
         "* greet: /greet",
         "    - utter_greet",
         "* goodbye: /goodbye",
+        "    - utter_goodbye",
```

Comment:
@wochinge Is this correct? For some reason the exported story had `utter_goodbye` as well as the last action but I am wondering why was it not there on the right hand side in the first place.

Reply:
I added a new story for goodbye. Previously `AugementedMemo` wouldn't know what to do and predict `action_listen`. When I decided that the `default_domain` should contain a `form`, I had to add the `RulePolicy`, which predicts a fallback in case it doesn't know what to do. To get everybody back on track I had to add story for `goodbye` so that `AugmentedMemoizationPolicy` can be in charge once more.

Reply:
Ahh okay so then it is needed indeed for this test. Thanks for the explanation!